rasa/shared/nlu/training_data/util.py

```diff
@@ -60,15 +71,17 @@ def get_file_format(resource_name: Text) -> Text:
     file_formats = list(map(lambda f: loading.guess_format(f), files))
 
     if not file_formats:
-        return "json"
+        return DEFAULT_FILE_FORMAT
+
+    knwon_file_formats = defaultdict(
```

Comment:
optional-ish: I actually had to look this up :-D I think it's simpler if it's just a regular dictionary and you use `dict.get(fformat, DEFAULT_FILE_FORMAT)` . Actually using a set would be even better, wouldn't it?

Reply:
yeah, its not needed. It was useful before but now it's useless